Sources/TidalAPI/Utils/TrackManifestParameters.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+/// Well-defined enums for trackManifest API parameters to prevent client errors
+public enum TrackManifestParameters {
+    
+    /// Manifest type for different platforms
+    public enum ManifestType: String, CaseIterable {
+        case hls = "HLS"            // iOS uses HLS
```

Comment:
To nitpick on the comments, web uses both HLS (for Safari) and DASH (for everyone else) :-)